install-trex.yaml

```diff
@@ -0,0 +1,6 @@
+---
+- hosts: trafficgen
```

Comment:
I think you also want the `remote_user` to be root to install the software given the default location is `/opt/trex`.